### PR TITLE
Development

### PR DESCRIPTION
**Pull request by theresenieva**
Create a basic homepage.

Created the Marker.java only to save the directories.

### CODE REVIEW

**Comment by theresenieva** [comment]
@googlebot I signed it!

**Review by vivianho18 — COMMENTED** [review]
LGTM 👍

File: prototype/src/main/java/com/google/sps/data/Marker.java [hunk 1]
```diff
@@ -0,0 +1 @@
+// Marker.java
```
Comment (anna-giang) [inline prototype/src/main/java/com/google/sps/data/Marker.java:1]: (Nitpick) Perhaps this file could have been added when the saving of the directories was actually implemented, because it is essentially an empty file, and seems to be unrelated to other modified files in this commit.

Comment (anna-giang) [reply prototype/src/main/java/com/google/sps/data/Marker.java:1]: Otherwise LGTM too! 😊

Comment (theresenieva) [reply prototype/src/main/java/com/google/sps/data/Marker.java:1]: True, I've split the commit into the two parts like you suggested.
